packages/next/src/build/swc/index.ts

```diff
@@ -25,6 +24,60 @@ const infoLog = (...args: any[]) => {
   Log.info(...args)
 }
 
+/**
+ * Based on napi-rs's target triples, returns triples that have corresponding next-swc binaries.
+ */
+const getSupportedArchTriples: () => Record<string, any> = () => {
+  const { darwin, win32, linux, freebsd, android } = platformArchTriples
+
+  return {
+    darwin,
+    win32,
```

Comment:
We don't do `ia32-gnu` or `x64-gnu` for `win32`, should those be filtered?

Reply:
wait why napi have *-gnu for win32 even 😭 

Thanks for the catch.

Reply:
For win32, explicitly picks up msvc as we'll never have other than msvc binaries.

https://github.com/vercel/next.js/pull/53230/files#diff-907b7be0cfc75bd37773e5ebc38d1deffa40861b8ac1cde92e4992e284a1ee59R37-R40